src/Interface/Others/Report/Results/index.js

```diff
@@ -279,6 +279,17 @@ class Results extends React.PureComponent {
               >
                 <img src={WipefestLogo} alt="Wipefest logo" style={{ height: '1.4em', marginTop: '-0.15em' }} /> Wipefest
               </a>
+              {' '}
+              {characterProfile && characterProfile.realm && characterProfile.name && characterProfile.region && (
+                <Link 
+                  to={`/character/${characterProfile.region.toUpperCase()}/${characterProfile.realm}/${characterProfile.name}/`} 
+                  data-tip={`Show ${characterProfile.realm} - ${characterProfile.name}'s most recent logs`}
```

Comment:
Show -> View (view is what the users wants to do, show is what the system would do)

logs -> reports (we call them reports usually in line with WCL's naming)